xaynetwork/xayn_discovery_app
# TB-3976 Push notifications deeplinks POC

### What 🕵️ 🔍

Initial work to allow for opening cards from push notifications. This is just a POC and some of the code will be updated when we have an ability to fetch a document from the engine.

----------

### How to test 🥼 🔬
- Feature flag enabled (pushNotificationsDeepLinks):
  - Go to Settings
  - Scroll down to the push notifications section
  - Tap on the Request button to request permission so that push notifications can be sent
  - Tap on the Send button to trigger a local notification (after 5 seconds) with the first loaded document

- Feature flag disabled:
  - The push notifications section in Settings should not be visible

----------

### References 📝 🔗

- [JIRA](https://xainag.atlassian.net/browse/TB-3976)

----------
